truffle/pom.rb

```diff
@@ -10,6 +10,7 @@
 
               'jruby.basedir' => '${basedir}/..' )
 
+  jar 'org.yaml:snakeyaml:1.14'
```

Comment:
Not sure if we should add a concrete dependency like this. JRuby seems to lazily add it to the class path or something, but not sure where it gets it from.